### DISCUSSION TITLE
How to remove duplicates and unwanted entries?

### DISCUSSION THREAD

**Discussion by Neobond** [moudey/Shell#329] [Q&A]
I found how to disable the More options menu for rotating images (very helpful since I do this often) but that has increased the size of the menu somewhat, now I want to remove some entries.

I want to remove the following:

- WinRAR (top one)
- Share with Skype
- Add to Favorites (what even is this?)
- Snagit

Is it possible to move some items like the bottom WinRAR and PowerRename into the same place as 7-Zip?

Thanks in advance

![SNAG-0002](https://github.com/moudey/Shell/assets/42996358/baf57edf-d653-4f9c-a029-5d3b6b2936f6)

**Comment by RubicBG** [comment]
- Which version of NS are you using?
- To remove duplicate items you need debug build 43 (v1.8.43) (https://discord.com/channels/1106387012707168318/1112126326665650176/1146552576381304882)
- unwanted items/menus can be hidden, can be removed (not loaded at all), can be moved to a sub-menu, or can be moved below - whichever way you want it?

**Reply by Neobond** [reply]
I am using 1.8.1 which is the latest official build I think.
